cclib/parser/data.py

```diff
@@ -169,8 +176,12 @@ class ccData:
        "scfenergies":      Attribute(numpy.ndarray,    'scf energies',                'optimization:scf'),
        "scftargets":       Attribute(numpy.ndarray,    'targets',                     'optimization:scf'),
        "scfvalues":        Attribute(list,             'values',                      'optimization:scf'),
+       "rotsymno":           Attribute(int,              'symmetry number',             'properties:rotational'),
+       "td_energy":        Attribute(list,             'TD energy',                   'optimization:TD'),
+       "td_energy_fin":    Attribute(float,            'TD energy',                   'properties:energy'),
        "temperature":      Attribute(float,            'temperature',                 'properties'),
        "time":             Attribute(numpy.ndarray,    'time',                        'N/A'),
+       "times_converged":  Attribute(numpy.ndarray,    'times converged',             'optimization'),

```

Comment:
```suggestion
       "freqdone":  Attribute(numpy.ndarray,    'done',             'vibrations'),

```

Reply:
this is a tricky point - "freqdone" sounds like the calculation included a frequency job. In Gaussian, even if the freq didn't converged to a stationary point you still get the frequency modes. Maybe "freqdone" could be "freqconv" or "freqconverged"?

Reply:
Hmm good point, and yes I agree the `freqconv` would probably be a better name. I suggested `freqdone` because we already have another attribute `optdone` which indicates optimisation convergence, so `freqdone` matches that.

If you prefer `freqconv` lets go with that, we can always look at updating `optdone` to a modern name in another PR in the future.

Reply:
ok, are you gonna include those changes or should I go ahead?